sernet.gs.service/src/sernet/verinice/model/common/CnATreeElement.java

```diff
@@ -268,8 +268,15 @@ public void remove() {
 		if (getParent() != null) {
 			getParent().removeChild(this);
 		}
-		
-		CopyOnWriteArrayList<CnALink> list2 = new CopyOnWriteArrayList<CnALink>(getLinksDown());
+		removeCnLinks();
+	}
+
+    /**
+     * Remove all links from and to this item.
+     * 
+     */
+    public void removeCnLinks() {
+        CopyOnWriteArrayList<CnALink> list2 = new CopyOnWriteArrayList<CnALink>(getLinksDown());
```

Comment:
What about a proper variable name, e.g. `linksDown`?

Reply:
It is just a refactored old code. But I am going to change it to.

Reply:
done